api/openapi/specs/api.yaml

```diff
@@ -119,6 +119,186 @@ paths:
           $ref: '#/components/responses/GetDataplaneXDSConfigResponse'
         '400':
           $ref: './common/error_schema.yaml#/components/responses/BadRequest'
+        '500':
+          $ref: '#/components/responses/Internal'
+  /meshes/{mesh}/dataplanes/{name}/_layout:
+    get:
+      operationId: get-dataplanes-layout
+      summary: Get networking layout for this dataplane
+      description: Returns detailed dataplane networking layout with most important information on dataplane and list of its inbounds and outbounds
+      parameters:
+        - in: path
+          name: mesh
+          required: true
+          description: The mesh of the DPP to get the layout for.
+          schema:
+            type: string
+        - in: path
+          name: name
+          required: true
+          description: The name of the DPP within the mesh to get the layout for.
+          schema:
+            type: string
+      responses:
+        '200':
+          $ref: '#/components/responses/DataplaneNetworkingLayoutResponse'
+        '400':
+          $ref: './common/error_schema.yaml#/components/responses/BadRequest'
+        '500':
+          $ref: './common/error_schema.yaml#/components/responses/Internal'
+  /meshes/{mesh}/dataplanes/{name}/_policies:
+    get:
+      operationId: get-proxy-policy-conf
+      summary: Get policies for this dataplane
+      description: Returns proxy level policy configuration derived from combining all proxy policies selecting this dataplane
+      parameters:
+        - in: path
+          name: mesh
+          required: true
+          description: The mesh of the DPP to get the diff for.
+          schema:
+            type: string
+        - in: path
+          name: name
+          required: true
+          description: The name of the DPP within the mesh to get the diff for.
+          schema:
+            type: string
+      responses:
+        '200':
+          $ref: '#/components/responses/ProxyPolicyConfResponse'
+        '400':
+          $ref: './common/error_schema.yaml#/components/responses/BadRequest'
+        '500':
+          $ref: './common/error_schema.yaml#/components/responses/Internal'
+  /meshes/{mesh}/dataplanes/{name}/_inbounds/{inbound-name}/_policies:
```

Comment:
why `inbound-name` and not `inbound-kri` as it says MADR?

Reply:
I've forgot to update it